workers/tasks/export/styles/printDocument.scss

```diff
@@ -1,19 +1,37 @@
-/* Editor Base Without Figures As Of 12/2019 */
-
-@import './editorBase.scss';
+/* Imports the equivalent of components/Editor/styles/base.scss,
+but without the media query mixin, which breaks PagedJS-cli:
+https://gitlab.pagedmedia.org/tools/pagedjs-cli/issues/11 */
+@import '~prosemirror-view/style/prosemirror';
+@import '~prosemirror-gapcursor/style/gapcursor';
+@import '~prosemirror-tables/style/tables';
+
+@import 'components/Editor/styles/latex.scss';
+
+.editor {
+    @import 'components/Editor/styles/main.scss';
+    @import 'components/Editor/styles/collab.scss';
+    @import 'components/Editor/styles/figure.scss';
+    @import 'components/Editor/styles/file.scss';
+    @import 'components/Editor/styles/table.scss';
+}
 
+/* Import some global variables and styles, including fonts */
 @import 'styles/variables.scss';
 @import 'containers/Pub/PubDocument/pubBody.scss';
```

Comment:
I have a proposed change to this strategy (that I'm happy to implement myself):

- Don't import `pubBody.scss` — instead, move the contents of its `pub-body-styles` mixin to `Editor/styles/main.scss` — where I think it ought to belong — and then remove the dummy `div.pub-body-component` wrapper from the export HTML? That would put all the CSS that's core to document presentation in one place and make the structure of the export documents easier to reason about from reading the code.

- Rather than writing all of the `.editor`-scoped imports out twice (except for the one having to do with figures), create a mixin for the entire set of editor styles (call it `editor-styles`) and use an `@if {}` block to conditionally import the content of `figure.scss`. _That_ file can then be written as bare styles rather than as a mixin.

- Finally, add a single `.editor { @include editor-styles(); }` in `Editor/styles/index.scss` and require that file from `Editor.js`.

Reply:
👍

Reply:
This ended up being a bit half-baked, as I forgot to account for the fact that not all Editor instances ought to take the pubBody styles, and it's apparently impossible to `@import` within a sass mixin. So I'm going to leave this alone.